entity/IntegrationEntities.xml

```diff
@@ -22,9 +16,22 @@
         <field name="shopifyOrderStatusId" type="text-short"/>
         <field name="shopifyFulfillmentOrderId" type="id"/>
     </extend-entity>
+
+    <!-- Shopify API returns a list of Fulfillment Orders per customer order -->
+    <entity entity-name="ShopifyFulfillmentOrder" package="mantle.shopify">
+        <field name="fulfillmentOrderId" type="id" is-pk="true"/>
+        <field name="orderId" type="id"/>
+        <field name="locationId" type="id">Shopify LocationId stored on Facility.externalId</field>
+        <field name="status" type="text-medium"/>
+    </entity>
+
     <extend-entity entity-name="OrderItem" package="mantle.order">
-        <field name="shopifyFulfillmentOrderLineItemId" type="id-long"/>
+        <field name="fulfillmentOrderId" type="id"/>

```

Comment:
Have you thought about using the `externalId` field instead of `fulfillmentOrderId` or does shopify's fulfillment orders have a different cardinality?

Reply:
there is no 'externalId' on the OrderItem entity